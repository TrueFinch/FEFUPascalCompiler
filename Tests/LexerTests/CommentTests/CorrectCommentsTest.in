//some single line comment
        { uuuu multiline comments are awesome



eeeeee}