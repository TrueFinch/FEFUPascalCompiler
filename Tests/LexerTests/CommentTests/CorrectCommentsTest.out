(1, 1)    | SingleLineComment   | //some single line comment    | //some single line comment    |
(2, 9)    | MultiLineComment    | { uuuu multiline comments are awesome\n\n\n\neeeeee}| { uuuu multiline comments are awesome\n\n\n\neeeeee}|